TUMFARSynchrony/experimental-hub
# Redux refactoring and introduce participant name

### Description
This PR modifies 
1) Folder structure of redux and hooks
2) Replace first_name and last_name of the participant with participant_name on frontend & backend

### Testing
1) run `npm install` and `npm run build`
2) Check there is only the participant name field is there while adding or editing participants.
3) Test the app flow from start to finish and make sure there are no issues.
